### ISSUE TITLE
Add transparent text editor

### ISSUE THREAD

**Issue by olivierkes** [olivierkes/manuskript#216]
The tendency nowadays seems to be towards apps as minimalists as possible in terms of interface. Every line that can be removed should be removed.

I've work quite hard to have an interface as simple as possible (though it's still often critiqued in reviews of manuskript), but I've now gone a step further.

Instead of having this:

![image](https://user-images.githubusercontent.com/1192933/32785244-605a71e8-c951-11e7-9799-db2bc62b832e.png)

You can now completely remove the border of the text editor, and have that:

![image](https://user-images.githubusercontent.com/1192933/32785420-cad30850-c951-11e7-8b11-4c34ed48883b.png)

It's not perfect yet, but it's a step closer.

Things that could be added:

- Option to remove the tab bar entierly
- Option to chose the background color (right now it's the default window application)

**Comment by gedakc** [comment]
Thank you @olivierkes for all your work on Manuskript.  You've been very productive as of late.

On the minimalist interface topic, I personally lean towards interfaces that are more obvious in what they do or support.  In my opinion, the top picture (original in 0.5.0) is much easier to determine what it does since you can type in the white box area and it is more obvious to me which tab is being typed in _Introduction_.

The bottom picture might look cleaner, but is much more confusing to me as to what I can do because the navigation tree blends into the writing area which blends into the tabs which blends into the buttons at the bottom ....

That's my 2 cents worth.  :-)

**Comment by hugo-id** [comment]
To me, minimalism refers to the _feeling_ of less.

In the first picture, you have the grey of the background, the white of the editable area, and the dark grey borders. In the second picture you have the grey of the background, and the dark grey of some(?) of the borders and you've had to add in _another_ colour to make up for the loss of feedback from removing the white background on the editable text.

There's equally as much visual clutter in both designs, and I would say the "new minimal" design _feels_ more cluttered.

And I'm now making mock-ups..
(My apologies)

![random_shit](https://user-images.githubusercontent.com/30918735/33429903-a8388832-d622-11e7-9a65-91b7f2eb80fd.png)

Haha, So as can be seen I've removed the borders, given the unselected document a darker shade of the background colour, and tweaked the options to the right (made them thinner, removed padding on left and right sides). I also came up with an A and B for potential ways to make them look even more modern, but that ties into my concern with their placement which option A highlights very well..

Why are they on the right side of the screen?

Whenever I first load the app (dev branch for typewriter following) I'm like "What do these do". I subsequently turn off the navigator, and forget which one's which. I belive that maybe a better location for these buttons is to place them closer to where they control things, like how it's been done in Android Studio,

![image](https://user-images.githubusercontent.com/30918735/33430181-ac1a930e-d623-11e7-8450-fb371671b087.png)

It's significantly busier, with far more visual elements on screen, but it is much easier to _intuit_. They ensure that the toggles are where you'd expect the element to be. So toggles for the various logs are on the bottom, visual elements and associated tools are on the right, with project navigation and major core functions controlled on the left.


There's a real chance I'm looking into this too much, but it's a part of my job, and I just love looking into this kind of thing! Once again, I absolutely love your app, think it's the best thing on the market (bar-none). Let me know if you're finding my critiques a little _too_ critical, or if there's somewhere else you'd like me to leave my feedback!
